build-tests/hashed-folder-copy-plugin-test/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import type { IRequireFolderOptions } from '@rushstack/hashed-folder-copy-plugin';
+
+declare function requireFolder(options: IRequireFolderOptions): string;
```

Comment:
I've never used this plugin before so I'm still understanding its functionality. Do you think we could add another source file imported by index.ts, that uses `requireFolder()` but also specifies different outputPath for the test case.

Reply:
Sure, added that.